docs/doxygen/config.doxyfile

```diff
@@ -38,7 +38,7 @@ PROJECT_NAME           = "coreJSON"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = "v1.0.0"
+PROJECT_NUMBER         = "V1.0.0"
```

Comment:
we are using a version string v1.0.0, why is this capitalized?

Reply:
Because it is capatalized in the license